Source/WebKit/UIProcess/API/wpe/qt6/WPEQtView.cpp

```diff
@@ -549,4 +585,32 @@ WebKitWebView* WPEQtView::webView() const
     return m_webView.get();
 }
 
+WPEQtProfile* WPEQtView::profile() const
+{
+    return m_profile;
+}
+
+void WPEQtView::setProfile(WPEQtProfile* profile)
```

Comment:
Remove, the `QWebEngineProfile` can only be set at construction time, so let us do the same. This would fit well with the intended usage of the `WebKitWebContext` API.